beacon-chain/slasher/detect_attestations_test.go

```diff
@@ -810,15 +801,11 @@ func BenchmarkCheckSlashableAttestations(b *testing.B) {
 		Slot:  &slot,
 	}
 
-	s := &Service{
-		params: DefaultParams(),
-		serviceCfg: &ServiceConfig{
-			Database:         slasherDB,
-			StateNotifier:    &mock.MockStateNotifier{},
-			HeadStateFetcher: mockChain,
-		},
-		attsQueue: newAttestationsQueue(),
-	}
+	s, _ := New(context.Background(), &ServiceConfig{
```

Comment:
```
beacon-chain/slasher/detect_attestations_test.go:804:5: Unhandled error for function call  (errcheck)
```

This error must be handled

Reply:
ok, will fix today. It was a mistake the other are handled

Reply:
done